### ISSUE TITLE
Preset a selected profile in the account drawer

### ISSUE THREAD

**Issue by Axxium** [mikepenz/MaterialDrawer#762]
Here is my issue i'm trying to set the account of a user upon adding on to the drawer the thing is i'm   getting the right account selected in the header but the drawer selected item stay the first one even though i used the setActiveProfile(). Is their a way to select the right account in the menu and in the header at the same time without clicking (Clicking on the item works perfectly)  ??

Thank you 

![genymotion - samsung galaxy s5 - 4 4 4 - api 19 - 1080x1920 1080x1920 480dpi - 169 254 38 101_2](https://cloud.githubusercontent.com/assets/7561038/10614118/7ff901d8-7726-11e5-96cf-ec094020a1c5.jpg)
![budgetpropane - hgit_3ejbudgetbudgetpropane - app - appsrcmainjavacombudgetpropanebudgetpropaneactivitiesdashboardactivity _2](https://cloud.githubusercontent.com/assets/7561038/10614120/821c5eba-7726-11e5-886c-869642cd46c7.jpg)

**Comment by mikepenz** [comment]
@Axxium i see that you use the position to select the active profile. Is it also your identifier of those items?

the `setActiveProfile` method takes an `identifier` as argument. not the position:
https://github.com/mikepenz/MaterialDrawer/blob/develop/library/src/main/java/com/mikepenz/materialdrawer/AccountHeader.java#L151

There is no method to define those things via position

**Comment by Axxium** [comment]
Yeah , mProfileCount is actualy the number of profile that i have and each time adding a profile im using this variable to define the drawer item identifier like this 
![budgetpropane - hgit_3ejbudgetbudgetpropane - app - appsrcmainjavacombudgetpropanebudgetpropaneactivitiesdashboardactivity _3](https://cloud.githubusercontent.com/assets/7561038/10614312/8d42b996-7727-11e5-95c5-34ba5cbf8910.jpg)

**Comment by Axxium** [comment]
So from my undestanding by using the method the account drawer item / header selected should be the same right ?

**Comment by mikepenz** [comment]
@Axxium yes it should.

when selecting that item. is the account selection opened?

**Comment by Axxium** [comment]
What do you mean by opened ? The drawer ?

**Comment by mikepenz** [comment]
yeah the drawer and the account selection. (so it shows the accounts in the drawer instead of the normal draweritems?)

**Comment by Axxium** [comment]
Yeah I have added a screenshot in the first message

**Comment by mikepenz** [comment]
yeah the screenshot does only show the problem :D but it can not show how it came to the problem. 

I will test and debug later. i think i just do not set the selection in the drawer. should be a quick fix. I'll comment here later

**Comment by Axxium** [comment]
Alright Thanks alot

**Comment by mikepenz** [comment]
a quick solution (until i have released the new version) for you would be something like this:

```java
if (mMenuHeader.isSelectionListShown()) {
   mDrawer.setSelection(profile.getIdentifier(), false);
}
```

**Comment by Axxium** [comment]
oki thx


On Tue, Oct 20, 2015 at 3:29 PM, Mike Penz <notifications@github.com> wrote:

> Closed #762 <https://github.com/mikepenz/MaterialDrawer/issues/762> via
> eb4a114
> <https://github.com/mikepenz/MaterialDrawer/commit/eb4a114d5ebc41d3e34d76d0282c256749536f91>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mikepenz/MaterialDrawer/issues/762#event-440738715>.
>



-- 
Martin Gauthier

H: 514-453-1749
C: 438-497-1749
